public/assets/css/style.css

```diff
@@ -282,9 +288,9 @@ body {
     color: black;
 }
 
-.SubmitButton {
+/*.SubmitButton {
     margin-bottom: 10vh;
-}
+}*/
```

Comment:
No dead code !

Reply:
Sorry... I've just removed it in the last commit